.vscode/settings.json

```diff
@@ -27,10 +27,8 @@
   "[javascript]": {
     "editor.formatOnSave": false
```

Comment:
I'd suggest turning this back on for now, I think the eol conversion is done with this auto formatter.